.editorconfig

```diff
@@ -0,0 +1,16 @@
+# top level config
+root = true
+
+# All files:
+# utf-8
+# Unix-style newlines
+# Remove end-of-file newlines
+# Soft indents (2 spaces)
+# Don't trim trailing whitespace
```

Comment:
Disagrees with line 16. Also, it's occasionally useful to be able to end a line with whitespace in Jade/Pug - I don't know if that's been done in this repo, but I've certainly found myself doing it.